tarantool_test.go

```diff
@@ -1549,6 +1580,113 @@ func TestClientNamed(t *testing.T) {
 	}
 }
 
+func TestClientRequestObjects(t *testing.T) {
+	var (
+		req  Request
+		resp *Response
+		err  error
+		conn *Connection
+	)
+
+	conn, err = Connect(server, opts)
+	if err != nil {
+		t.Errorf("Failed to connect: %s", err.Error())
+		return
+	}
+	if conn == nil {
+		t.Errorf("conn is nil after Connect")
+		return
+	}
+	defer conn.Close()
+
+	// The code prepares data.
+	for i := 1010; i < 1020; i++ {
+		_, err = conn.Replace(spaceName, []interface{}{uint(i), fmt.Sprintf("val %d", i), "bla"})
+		if err != nil {
+			t.Errorf("Failed to Replace: %s", err.Error())
+		}
+	}
+
+	// Update without operations.
```

Comment:
Is it legal? What we'll get in the response? Is it worth to tests responses (as I see, we only test response for a select request)?

Reply:
I think it is legal until a tarantool instance does not return an error. It returns unmodified tuple.

I've added checks for responses.

Reply:
Okay. It seems, it is actually legal. I didn't know.